src/components/Layout/SelectLang/index.js

```diff
@@ -11,7 +11,7 @@ class SelectLang extends PureComponent {
     const domain = window.location.hostname;
     const languages = [{ key: 'en', code: '🇬🇧', name: 'English' }];
     // Only on DEV server or locally we should see fake translations
-    if (domain.endsWith('gbif-dev.org') || domain.startsWith('rtl') || domain === 'localhost') {
+    if (domain.endsWith('gbif-dev.org') || domain.endsWith('netlify.com') || domain === 'localhost') {
```

Comment:
Please remove this logic from the component and move it to a general config (such as the one where APIs in configured). And include test locales for anything but uat and prod.